src/component/page/LoggetUtPage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+// Laster tom side for å unngå videre navigasjon etter at token har utgått
+// SesjonNotifikasjon alertstripe vil vises øverst
+export const LoggetUtPage = () => {
+	return <div/>
```

Comment:
Man kan ikke lukke den alert stripen sant? Kunne man lagd en "Du er nå logget ut, trykk her for å logge inn igjen" Side og droppe alertstripen? For meg hadde det gitt litt mer mening, siden om jeg hadde sett denne siden så måtte jeg lest kommentaren for å forstå hva som skjer.

Reply:
Ja, enig.. Ble litt rart med tom side. Det ville blitt en litt rar greie(som kanskje ikke ville merkes uansett) med at man må fjerne alertstripen før man redirecter til den nye siden. Pusher nå for å se om alarmene roer seg men tar med tilbakemeldingen til neste steg.

Om redirect urlen så ser jeg at det ble omvendt så har fiksa. Men grunnen til at man redirecter til gjennomføringPage er at når alertstripen dukker opp så kan det hende man står på LoggetUtPage som da ville ført til at man blir logget inn og får en forvirrende tom side. Det kan evt løse med å lage den siden som du foreslår siden man da kan spesialhåndtere redirect url i LoggetUtPage.